livekit-agents/livekit/agents/worker.py

```diff
@@ -118,6 +118,8 @@ class WorkerOptions:
     """Maximum amount of time to wait for a process to initialize/prewarm"""
     permissions: WorkerPermissions = field(default_factory=WorkerPermissions)
     """Permissions that the agent should join the room with."""
+    agent_name: str = "default"
```

Comment:
is this the right default to have? wondering how the server code is treating it, maybe there is no distinction